charts/argo-cd/values.yaml

```diff
@@ -2336,10 +2332,6 @@ notifications:
   # -- Labels to be applied to the controller Pods
   podLabels: {}
 
-  # -- Pod Security Context
-  securityContext:
```

Comment:
Removed to make it consistent with the rest of components that use only global security context